comp/remote-config/rcclient/rcclient.go

```diff
@@ -86,11 +87,26 @@ func newRemoteConfigClient(deps dependencies) (provides, error) {
 		return provides{}, err
 	}
 
+	var clientHA *client.Client
+	if config.Datadog.GetBool("ha.enabled") {
+		clientHA, err = client.NewUnverifiedHAGRPCClient(
+			ipcAddress,
+			config.GetIPCPort(),
+			security.FetchAuthToken,
+			client.WithAgent("unknown", version.AgentVersion), // We have to create the client in the constructor and set its name later
+			client.WithPollInterval(5*time.Second),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏Do we know if we should change this for HAMR?

Reply:
Good question. I would imagine we want to make the interval as large as possible to reduce load on R2 while still meeting any SLOs for how quickly we expect the failover trigger to be effective. I can ask around if we have any such expectations/SLOs

Reply:
Discussed offline. For MVP, this is fine using the same default configs as the R1 rc service. In a future change, we may make it independently configurable with different default values.